modules/webbrowser/src/processors/webbrowserprocessor.cpp

```diff
@@ -86,7 +86,10 @@ WebBrowserProcessor::WebBrowserProcessor(InviwoApplication* app)
     };
     updateVisibility();
 
-    auto reload = [this]() { browser_->GetMainFrame()->LoadURL(getSource()); };
+    auto reload = [this]() {
+        isLoading_ = true;
```

Comment:
If the state changing does not work as expected we should add a comment that this is a workaround

Reply:
Another thing coming to my mind: CEF handles that stuff asynchronously meaning that the OnLoadingStateChange callback can potentially be very late in contrast to setting the property directly.

Reply:
As long as it can be guaranteed that the onloadingstatechanged function is eventually called it should be fine. Otherwise, this change might make the processor indicate that it is loading even though it never was

Reply:
why? `reload()` is triggered, hence the state change. If CEF decides that's not worth a reload the loading state change callback should reflect this. And in case it doesn't then the processor is still officially in a "loading" state.